iocore/net/Connection.cc

```diff
@@ -229,7 +229,7 @@ Server::setup_fd_for_listen(bool non_blocking, const NetProcessor::AcceptOptions
 #endif
   }
 
-  if ((opt.sockopt_flags & NetVCOptions::SOCK_OPT_NO_DELAY) &&
+  if ((opt.sockopt_flags & NetVCOptions::SOCK_OPT_NO_DELAY) && !opt.f_mptcp &&
       safe_setsockopt(fd, IPPROTO_TCP, TCP_NODELAY, SOCKOPT_ON, sizeof(int)) < 0) {
     goto Lerror;
   }
```

Comment:
Can you help me understand this change? I assume TCP_NODELAY is not applicable to MPTCP?

Reply:
It seems like now that we are using MPTCP as a protocol and not a socket option, the socket options for IPPROTO_TCP doesn't apply for sockets that are not defined for IPPROTO_TCP